src/main/webapp/style.css

```diff
@@ -19,8 +18,8 @@
     border: thin solid black;
     height:100px;
```

Comment:
make sure the height and width in css styling matches the chart's javascript to prevent the chart from overlaying any of our text boxes